layers/+source-control/git/packages.el

```diff
@@ -205,10 +204,9 @@ Press [_b_] again to blame further in the history, [_q_] to go up or quit."
     :if git-enable-magit-svn-plugin
     :commands turn-on-magit-svn
     :init (add-hook 'magit-mode-hook 'turn-on-magit-svn)
-    :config
-    (progn
-      (evil-define-key 'emacs magit-status-mode-map
-        "N" 'magit-key-mode-popup-svn))))
+    :config (progn
+              (spacemacs|diminish magit-svn-mode "SVN")
+              (define-key magit-mode-map "!" 'magit-svn-popup))))
```

Comment:
Currently `!` is bound to `magit-run-popup`. Is there any other key you can think of to replace this?